docstore/azblob/azblob.go

```diff
@@ -206,3 +206,28 @@ func (s *Store) Put(ctx context.Context, key string, body []byte) error {
 
 	return nil
 }
+
+func deleteBufFromBlob(ctx context.Context, blobURL azblob.BlockBlobURL) error {
```

Comment:
Is there an advantage to splitting this out into a separate function?

Reply:
Not particularly, it's just in-line with the existing formatting but I can reincorporate it